spec/aboutApp.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { shallow, ShallowWrapper } from 'enzyme';
+import * as React from 'react';
+import AboutApp from '../src/renderer/components/about-app';
+
+describe('about app', () => {
+    const { ipcRenderer } = require('./__mocks__/electron');
+    const aboutAppDataLabel: string = 'about-app-data';
+    const onLabelEvent: string = 'on';
+    const removeListenerLabelEvent: string = 'removeListener';
+
+    it('should render correctly', () => {
+        const wrapper: ShallowWrapper = shallow(React.createElement(AboutApp));
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should call `about-app-data` event when component is mount', () => {
```

Comment:
typo. sentence should be `should call about-app-data event when component is mounted`